build/python_metadata.bzl

```diff
@@ -0,0 +1,5 @@
+# The below is a list of pyodide-lock.json files for each package bundle version that we support.
+# Each of these gets embedded in the workerd and EW binary.
+PYTHON_LOCKFILES = {
+    "20240829.4": "c2d9c67ea55a672b95a3beb8d66bfbe7df736edb4bb657383b263151e7e85ef4",
```

Comment:
What is this value here? Presumably we should update the package automation to put it in the automatically generated `pyodide_bucket.bzl` and import it from there.

Reply:
I'll add a comment to explain what the value is but it is just the sha256 hash of the lock file.

Do we want the package automation to enumerate all the old package bundle releases so it can generate the full list of package dates here?